minesweeper.py

```diff
@@ -342,7 +343,7 @@ def get_avg_time(highscore_dict):
                 f"{int(highscore_dict['time'] / 60)} min & {highscore_dict['time'] % 60} sec"
             )
         else:
-            format_time = f"{highscore_dict['time']} sec"
+            format_time = f"{int(highscore_dict['time'])} sec"
```

Comment:
Also change the `if` part, which is used for games that last more than one minute.

To keep the code clean, maybe make a variable, `seconds = int(highscore_dict['time'])` or something like that.